modules/commission.py

```diff
@@ -81,7 +107,7 @@ def completed_handler(self):
                 self.stats.increment_commissions_received()
                 continue
             if Utils.find("menu/item_found"):
-                Utils.touch_randomly(self.region["tap_to_continue"])
+                Utils.find_and_touch("menu/tap_to_continue")
```

Comment:
Why?

Reply:
Because if we receive many items the position of the button will change, I have not had problems with it so far, because a click anywhere on the screen makes the popup close, but I think this implementation more reliable.

Reply:
Ohhhh, that makes sense. I will check it out later, thanks!